platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/FutureEventBuffer.java

```diff
@@ -112,8 +112,13 @@ public List<GossipEvent> addEvent(@NonNull final GossipEvent event) {
     public List<GossipEvent> updateEventWindow(@NonNull final NonAncientEventWindow eventWindow) {
         this.eventWindow = Objects.requireNonNull(eventWindow);
 
+        // We want to release all events with birth rounds less than or equal to the pending consensus round.
+        // In order to do that, we tell the sequence map to shift its window to the oldest round that we want
+        // keep within the buffer.
```

Comment:
grammar: should be either "kept" or "to keep"

Reply:
Switched to `to keep`